src/plist.c

```diff
@@ -220,8 +221,7 @@ Obj TYPE_LIST_EMPTY_IMMUTABLE;
 Obj TYPE_LIST_HOM;
 
 #define IS_TESTING_PLIST(list) \
-    (FIRST_TESTING_TNUM <= TNUM_OBJ(list) \
-  && TNUM_OBJ(list) <= LAST_TESTING_TNUM)
+    (IS_BAG_REF(list) && TEST_OBJ_FLAG(list, TESTING))
```

Comment:
I don't think we need to; but this code is copied verbatim from the HPC-GAP side, and as usual, I tried to refrain from any changes beyond that as much as possible, to reduce the risk of regressions.